tests/test_coreir/test_operator.py

```diff
@@ -1,166 +1,8 @@
-import pytest
 import magma as m
-import mantle
 from magma.testing import check_files_equal
-from collections import namedtuple
 import fault
 from hwtypes import BitVector
 
-op = namedtuple("op", ["name", "operator"])
```

Comment:
Are these tests replicated in magma?

Reply:
Yup, https://github.com/phanrahan/magma/pull/515/files#diff-91664f982686979fc326ee1257287298R147-R308